src/main/java/io/gingersnapproject/cdc/cache/hotrod/HotRodCacheBackend.java

```diff
@@ -83,6 +83,20 @@ public void start() {
       stopped = false;
    }
 
+   @Override
+   public boolean reconnect() {
+      if (!stopped) return true;
```

Comment:
This looks possibly broken. If the start ever receives an exception it will be marked as stopped and never retried again. Is this handled some place else I am not seeing?

Reply:
I'll need to remove a small piece, but ultimately this is called within the `ReschedulingTask`.

Reply:
If somehow stopped is ever marked to true then it will never reconnect, `ReschedulingTask` just calls this method which will do nothing. I don't think this is correct still. However, it looks like this code is reducing how many starts are invoked on the remote cache as the old code appears to be invoking it twice.

Either way I think maybe we should revisit this in a separate PR with a reconnection test anyways.